src/dialogs/ColumnPicker.js

```diff
@@ -22,8 +22,11 @@ var ColumnPicker = Dialog.extend('ColumnPicker', {
         this.grid = grid;
 
         if (behavior.isColumnReorderable()) {
-            // grab the lists from the behavior
-            if (behavior.setGroups){
+            // parse & add the drag-and-drop stylesheet addendum
+            var stylesheetAddendum = stylesheet.inject('list-dragon-addendum');
+
+            // grab the group lists from the behavior
+            if (behavior.setGroups) {
```

Comment:
I would add
```
+            //TEMPORARY WORKAROUND. Column Picker will become a add-on with a noted dependency on group view
+            if (behavior.getGroups && behavior.getGroups().length > 0) {
                 this.selectedGroups = {
```
If groupView has been mixed in but the groups have been set to `[]`